src/content_scripts/common.js

```diff
@@ -8,6 +8,9 @@
 let functionalMode = null;
 let fakedColorStatus = null;
 
+// the color status last shown to the page javascript
+let jsColorStatus = fakedColorStatus;
```

Comment:
Hmm? Why do you need to save it two times? Is not `fakedColorStatus` enough?

Reply:
Oops, that is a leftover thing from earlier experiments.